### PR TITLE
Some fixes for topics in Getting Started and Developing/API

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md [hunk 1]
```diff
@@ -103,4 +103,4 @@ dependencies {
 You must reference `net.corda:corda-crypto-extensions`.
 {{< /note >}}
 
-For the CorDApp packaging instructions, see the relevant documentation.
+For the CorDApp packaging instructions, see the section on [CordApp Packaging](../tutorials/packaging.htm).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md:106]: That tutorial section isn't published for DP2

Comment (nadinequinr3) [inline content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md:106]:
```suggestion
For the CorDapp packaging instructions, see the section on [CordApp Packaging](../tutorials/packaging.htm).
```

File: content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md [hunk 2]
```diff
@@ -10,20 +10,20 @@ section_menu: corda-5-dev-preview2
...
+Currently there is a time-out issue when debugging Corda. As the local Corda cluster is a real Corda cluster, it behaves like one. Specifically, if the workers stop working, Corda detects it and errors out the flow. Unfortunately, it currently mistakes code held at a debugging point as a processor that has stopped working. As a result, although you can halt at a breakpoint and examine the stack, if you wait too long before resuming the execution of the code, it times out and the CorDapp does not continue to run as expected.     
 {{< /note >}}
 To debug:
 1. [Start the Corda combined worker using the startCorda Gradle helper](../running-your-first-cordapp/run-first-cordapp.html#starting-the-corda-combined-worker), if not already started.
 2. [Deploy your CorDapp using deployCordapp](../running-your-first-cordapp/run-first-cordapp.html#deploying-a-cordapp), if not already deployed.
 3. Select the `DebugCorDapp` run configuration.
 {{< figure src="select-debug-configuration.png" figcaption="CSDE DebugCorDapp run configuration in IntelliJ" >}}
-4. Click the debug button.
+4. Click the **Debug** button.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md:20]: Should this be bold even though it's not actually written on the interface?

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md (hunk 2 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md:20]: Sorry happy for it to be bold if that makes sense to you. More a point for discussion at some point

File: content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md:106]: Ah ok! Please revert that.

File: content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md:20]: I generally think the title of a part of the interface should be bolded as per the style guide even to make it stand out but yeah can add it to things to discuss.

File: content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md]: OK reverted that change and pushed.

**Review by nadinequinr3 — DISMISSED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**